core/ir/aarch64/instr_create_api.h

```diff
@@ -3376,4 +3376,11 @@
  */
 #define INSTR_CREATE_fmov_scalar_imm(dc, Rd, f) instr_create_1dst_1src(dc, OP_fmov, Rd, f)
 
+/**
+ * Creates a XPACI instruction.
+ * \param dc      The void * dcontext used to allocate memory for the instr_t.
+ * \param Rd      Register with PAC bits to remove.
+ */
+#define INSTR_CREATE_xpaci(dc, Rd) instr_create_0dst_1src((dc), OP_xpaci, (Rd))
```

Comment:
Where are all the other new opcodes added?  Though xref #2440 and long-term these should be auto-generated.

Reply:
`codec.py` will autogenerate the new opcodes from the instr definitions in `codec_v83.txt` and add to the `opcodes_*.h` files, placed in the build directory.